components/tag/index.tsx

```diff
@@ -106,24 +108,39 @@ class Tag extends React.Component<TagProps, TagState> {
   }
 
   renderTag = (configProps: ConfigConsumerProps) => {
-    const { children, ...otherProps } = this.props;
+    const { children, icon, ...otherProps } = this.props;
     const isNeedWave =
       'onClick' in otherProps || (children && (children as React.ReactElement<any>).type === 'a');
     const tagProps = omit(otherProps, ['onClose', 'color', 'visible', 'closable', 'prefixCls']);
+    warning(
+      !(typeof icon === 'string' && icon.length > 2),
+      'Tag',
+      `\`icon\` is using ReactNode instead of string naming in v4. Please check \`${icon}\` at https://ant.design/components/icon`,
+    );
+    const iconNode = icon || null;
+    const kids = iconNode ? (
+      <>
+        {iconNode}
+        <span>{children}</span>
+      </>
+    ) : (
+      children
+    );
```

Comment:
So is this.

Reply:
Sorry, not sure what u mean

Reply:
@afc163 🔝

Reply:
We don't need this either, cause antd v3 never supported Tag[icon].

Reply:
Oh, that what you meant :) Fixed. Thought you somehow related to the rebase thing